content/training/Walk-through/template_setup.Rmd

```diff
@@ -119,17 +120,17 @@ Set the epidemiological week for the report. **This value is used in later code
 
 By default in the templates epi weeks begin on Mondays (this can be changed at the bottom of the "setup" code chunk). See the documentation of the [package "aweek"](https://cran.r-project.org/web/packages/aweek/vignettes/introduction.html) for more information. 
 
-To determine which epi week of a date you can use the date2week() function. Let's assume the date of this analysis is 20 April, 2017.
+To determine which epi week of a date you can use the date2week() function. It will tell you the year, week, and day number of the provided date. Let's assume the date we want to use for this analysis is 20 April, 2017.
```

Comment:
```suggestion
To determine which epi week of a date you can use the `date2week()` function. It will tell you the year, week, and day number of the provided date. Let's assume the date we want to use for this analysis is 20 April, 2017.
```